cm2350/intc_exc.py

```diff
@@ -723,28 +744,12 @@ class MceWriteBusError(MachineCheckException):
     # Set the MCSR[ST] bit
     __mcsrbits__ = 0x00004000
     __mcsrmask__ = 0x00004000
-    
-class BreakException(DebugException):
-    __SIGNAL__ = None
-
-    def setupContext(self, emu):
-        # For now, we'll skip the standard setupContext and attempt to handle 
-        # "Breaks" out of the standard interrupt handling context.
-
-        # use the emulator's built-in pause/resume functionality (so we're not 
-        # left dealing with the details here)
-        emu._do_halt(self.__SIGNAL__)
-
-class SigTRAP_Exception(BreakException):
-    __SIGNAL__ = signals.SIGTRAP
-
-class SigSTOP_Exception(BreakException):
-    __SIGNAL__ = signals.SIGSTOP
 
-class SigTSTP_Exception(BreakException):
-    __SIGNAL__ = signals.SIGTSTP
```

Comment:
What happened to these classes?

Reply:
I realized I was starting to take the debug peripheral emulation too far in a way that didn't make sense yet.

Until we want to try and emulate the capabilities of a trace32.